pkg/xds/context/mesh_context_builder.go

```diff
@@ -161,6 +162,11 @@ func (m *meshContextBuilder) BuildIfChanged(ctx context.Context, meshName string
 		dataplanesByName[dp.Meta.GetName()] = dp
 	}
 	meshServices := resources.MeshServices().Items
+	meshExternalServices := resources.MeshExternalServices().Items
+	meshExternalServicesByName := make(map[string]*meshextenralservice_api.MeshExternalServiceResource, len(dataplanes))
```

Comment:
Isn't `len(dataplanes)` too much, or simply wrong? I don't think that number of mes will correlate with number of ddps

Reply:
I think @lukidzi just copied it from a couple of lines above. Will remove it.